### ISSUE TITLE
Cover more cases in Layout/MultilineMethodCallIndentation

### ISSUE THREAD

**Issue by bquorning** [rubocop/rubocop#11513]
I was surprised to learn that `Layout/MultilineMethodCallIndentation` does _not_ complain about the following code (despite very clearly inconsistent indentation).

```rb
# frozen_string_literal: true

def foo
  bar(
    key: VeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeryLongClassName
 .veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeery_long_method_name
  )
end
```

However, it seems to have been the case ever since the cop was introduced in 5cf70902becb2b5db2c60482f84b4e3fc4d19195.

Digging a bit further, it seems that the behavior was introduced (475d652f4e35a6f085d6bd2999b623036155749d) as a change to `Style/MultilineOperationIndentation` in response to a bug report (#1362) filed by myself 😅

The decision to ignore this node happens when `#inside_arg_list_parentheses?` returns true here: https://github.com/rubocop/rubocop/blob/6e5fc10964a6bc7e06c1e8cf62f16ab489d0a89b/lib/rubocop/cop/mixin/multiline_expression_indentation.rb#L197-L201

## Describe the solution you'd like

```rb
# EnforcedStyle: aligned
def foo
  bar(
    key: VeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeryLongClassName
    .veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeery_long_method_name
  )
end

# EnforcedStyle: indented_relative_to_receiver
def foo
  bar(
    key: VeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeryLongClassName
           .veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeery_long_method_name
  )
end

# EnforcedStyle: indented
def foo
  bar(
    key: VeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeryLongClassName
      .veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeery_long_method_name
  )
end
```

**Comment by bquorning** [comment]
This issue is still relevant.

**Comment by AlexWayfer** [comment]
Still actual…
